wrappers/Python/pytest/test_refprop_alias_resolution.py

```
@@ -0,0 +1,402 @@
+def _cp_rp_index() -> dict[str, tuple[str, str, list[str]]]:
+    """
+    Return {cp_canonical: (rp_name, cp_canonical, aliases)} for every CoolProp
+    fluid whose REFPROP_NAME appears in the FLUIDS directory.
+    """
+    available = _fld_stems() | _ppf_stems()
+    index: dict[str, tuple[str, str, list[str]]] = {}
+    for cp_name in CP.get_global_param_string("FluidsList").split(","):
+        try:
+            info = json.loads(CP.get_fluid_param_string(cp_name, "JSON"))[0]["INFO"]
+        except Exception:
+            continue
```

Ruff (0.15.15)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)